scripts/us_bea/states_gdp/test_import.py

```diff
@@ -21,6 +21,9 @@
 import unittest
 import pandas as pd
 from import_data import StateGDPDataLoader
```

Comment:
Per https://google.github.io/styleguide/pyguide.html#22-imports, we import modules, not classes within modules.  This eases determining what items are imported.